src/api/app/helpers/webui/notification_helper.rb

```diff
@@ -37,4 +111,11 @@ def notification_filter_active?(filter_item)
   def badge_color(filter_item)
     notification_filter_active?(filter_item) ? 'badge-light' : 'badge-primary'
   end
+
+  def reviewers_tags(review)
+    return user_avatar(User.find_by(login: review.by_user)) if review.by_user
+    return user_avatar(Group.find_by(title: review.by_group)) if review.by_group
+    return simulated_avatar('fa-archive text-warning', "Package #{review.by_project}/#{review.by_package}") if review.by_package
+    return simulated_avatar('fa-cubes text-secondary', "Project #{review.by_project}") if review.by_project
```

Comment:
Review by package and review by project mean users too. You could fetch them from the `Review.users_and_groups_for_review`.

Reply:
If I'm not wrong, the idea here is to replace this "simulated avatar":
![Screenshot_2020-07-10 Open Build Service](https://user-images.githubusercontent.com/2581944/87148418-a1a0f600-c2ae-11ea-8c01-4c2bf3877cbd.png)
by the avatars of the users and groups related to that project.

This can end up with a huge list of avatars that are going to be hidden anyway because we are displaying only 6.

So if we keep what we have and show the "simulated avatar" (display project/package icon) we sum up what we want to say: "_Hey you, people involved in this project/package. Have a look!"_. And there is a high probability they are visible. In the other way around, we'll just see a couple of new faces only.

Reply:
Okay we can also postpone this and find the middle ground. I'm fine with creating something in the backlog

Reply:
You or me?

Reply:
I created it https://trello.com/c/HMSq1L3N